compiler/rustc_macros/src/diagnostics/fluent.rs

```diff
@@ -81,217 +38,215 @@ fn invocation_relative_path_to_absolute(span: Span, path: &str) -> PathBuf {
     }
 }
 
+/// Tokens to be returned when the macro cannot proceed.
+fn failed() -> proc_macro::TokenStream {
+    quote! {
+        mod fluent_generated {
+            pub static DEFAULT_LOCALE_RESOURCE: &'static str = "";
```

Comment:
The `mod fluent_generated` wrapper should be removed I think.

Reply:
Do you think there should still be a wrapper but just external to the macro? I'd be okay with that as an alternative.

Reply:
`DEFAULT_LOCALE_RESOURCE` in the non-error case is put at the crate root, not in `fluent_generated`. The error case should match this to prevent cascading errors as intended.

Reply:
Ah, I see, fixed.